frontend/src/services/cartService.ts

```diff
@@ -0,0 +1,62 @@
+import { Storage } from '../utils/localStorage';
+
+export namespace CartService {
+  let cart = Storage.getItem(Storage.keys.cart) || [];
+  let listeners = [];
+
+  export const syncWithLS = () => {
```

Comment:
syncWithLS -> syncWithLocalStorage
syncWithLS shouldn't be exported

Reply:
would be better to rename to updateCart, with argument newCartData and body cart = newCartData and updating of localStorage